test/src/xquery/xquery3/fn.xql

```diff
@@ -34,23 +34,28 @@ function fnt:has-children-contextItem-noChildren() {
     <a/>/has-children()
 };
 
+(: ERROR: It is a dynamic error if evaluation of an expression relies on some part of
+          the dynamic context that has not been assigned a value. Context item is absent
 declare
     %test:assertFalse
 function fnt:has-children-contextItem-empty() {
     ()/has-children()
 };
+:)
 
 declare
     %test:assertError("XPDY0002")
 function fnt:has-children-contextItem-absent() {
     has-children()
 };
 
+(: fail with XPTY0019
```

Comment:
the problem that `fn.xql` silently ignored right now because of [this](https://github.com/eXist-db/exist/pull/1285/commits/726ca3407f94cf2c7bfd44959223a066bc744607#diff-07e7d69906fa424961c02b9044da8db4R96) and [that](https://github.com/eXist-db/exist/pull/1285/commits/726ca3407f94cf2c7bfd44959223a066bc744607#diff-07e7d69906fa424961c02b9044da8db4R122).

And after it run this test fail because actual error code is different.

To fix root issue I commit [this](https://github.com/eXist-db/exist/pull/1285/commits/726ca3407f94cf2c7bfd44959223a066bc744607#diff-8306b9c569c683640a7bdb63c01cfae1R71) change.